<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>quan_li_dai_li</class>
 <widget class="QDialog" name="quan_li_dai_li">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>872</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sửa/Xóa đại lý</string>
  </property>
  <widget class="QLineEdit" name="le_tukhoaDL">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_thoat_dl">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>340</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Thoát</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>361</width>
     <height>231</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Đại lý</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListView" name="listView"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pb_sua_dl">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Sửa</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_timkiem_DL">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>40</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Tìm kiếm</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_xoa_dl1">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>340</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Xóa</string>
   </property>
  </widget>
  <widget class="QFrame" name="frm_sua_dai_li">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>40</y>
     <width>371</width>
     <height>371</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>MaDL</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TenDL</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Dia_chi</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>SDT</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Email</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_maDL">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_tenDL">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>90</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_dia_chiDL">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>130</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_sdtDL">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>170</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_email_DL">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>210</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_sua_dl_save">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>330</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Lưu</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
